tests/builder/app_settings.rs

```diff
@@ -137,6 +137,9 @@ fn arg_required_else_help_with_default() {
     );
 }
 
```

Comment:
This test shows a weakness of the current simple implementation -- the end user is able to extract the text from the error message without calling the print method, which is where I currently carry out interpolation. Will need to find a more ideal place to put that logic so that users can never extract text from Clap that hasn't been interpolated.